### ISSUE TITLE
Port it to Python or NodeJS

### ISSUE THREAD

**Issue by steelbrain** [k2b6s9j/cli-linter-generator#1]
Python preferably, 'cause almost all of the people have python installed by default

**Comment by steelbrain** [comment]
I can help with it, if you want :)

**Comment by AsaAyers** [comment]
1. Why would I install a binary for something I use once to generate a linter? It should be a script I can run easily. Preferably in my own language
2. Your target audience isn't able to help if you won't write the tool in their language.

Since this generates ES6, it should definitely be written in ES6

**Comment by steelbrain** [comment]
@AsaAyers You have my :+1:

**Comment by AsaAyers** [comment]
Atom has a package generator, and minimap has a set of generators:

![selection_147](https://cloud.githubusercontent.com/assets/324999/8761659/5e0fef14-2d2a-11e5-8545-b7036a1b53bb.png)

If it's not too complex maybe we should do the same thing in `linter`. What do you think?

**Comment by steelbrain** [comment]
Does it have a public API?

**Comment by AsaAyers** [comment]
¯\_(ツ)_/¯ I've done seconds if research (i searched for generate in the command list)

**Comment by AsaAyers** [comment]
I don't think so. For some reason minimap uses a [custom element](https://github.com/atom-minimap/minimap/blob/master/lib/main.coffee#L144) and attaches it to atom and it generates the project. It's probably more effort than it's worth since we don't really need to generate a completely new project.
